go.mod

```diff
@@ -3,11 +3,16 @@ module github.com/desmos-labs/desmos/v4
 go 1.19
 
 require (
+	cosmossdk.io/api v0.3.1
+	cosmossdk.io/errors v1.0.0-beta.7
+	cosmossdk.io/math v1.0.0
 	github.com/CosmWasm/wasmd v0.40.0-rc.0
 	github.com/armon/go-metrics v0.4.1
 	github.com/btcsuite/btcd/btcec/v2 v2.3.2
+	github.com/cometbft/cometbft v0.37.1
+	github.com/cometbft/cometbft-db v0.8.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.2
-	github.com/cosmos/cosmos-sdk v0.47.1
+	github.com/cosmos/cosmos-sdk v0.47.2
```

Comment:
It is would be better to upgrade desmos-cosmos-sdk to 0.47.2 as well.

Reply:
Thanks, updated it!